components/Layout/Header/index.tsx

```diff
@@ -58,7 +48,7 @@ export function Header () {
 
         <HeaderNavigation />
 
-        <ToggleThemeButton marginLeft={[2, null, 'unset']} />
+        {/* <ToggleThemeButton marginLeft={[2, null, 'unset']} /> */}
```

Comment:
Needs to add a comment on this component since currently there's no toggle functionality from `chakra-ui-flashless`

Reply:
Will implement it back when the toggle functionality from `chakra-ui-flashless` is done